src/main/java/homework10/Util.java

```diff
@@ -0,0 +1,72 @@
+package homework10;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Util {
+     public static Logger logger = Logger.getLogger(Util.class.getName());
+
+    public static void main(String[] args) {
+        List<Map<List<Set<Integer>>, String>> list = new ArrayList<Map<List<Set<Integer>>, String>>();
+        Set<Integer> set = new HashSet<Integer>();
+        set.add(5);
+        set.add(6);
+
+        List<Set<Integer>> setList = new ArrayList<Set<Integer>>();
+        setList.add(set);
+
+        Map<List<Set<Integer>>, String> map = new HashMap<List<Set<Integer>>, String>();
+        map.put(setList, "Slava");
+        map.put(setList, "Cheban");
+
+        list.add(map);
+        try {
+            Util.wrightListToXML(list);
+        } catch (IOException e) {
+            logger.info("something not right");
+        }
+    }
+
+    public static void wrightListToXML(List list) throws IOException{
+        XmlMapper xmlMapper = new XmlMapper();
+        FileWriter writer = new FileWriter("writeXMLHere.xml");
+        xmlMapper.writeValue(writer, list);
+    }
+
+    public static boolean checkPhoneNumber(String number) {
```

Comment:
better move validation methods to the separate class
remember about SOLID principles